Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/OperatorEntry.kt

```diff
@@ -0,0 +1,400 @@
+package org.cqframework.cql.cql2elm.model
+
+import java.util.*
+import kotlin.collections.ArrayList
+import kotlin.collections.HashMap
+import org.hl7.cql.model.ChoiceType
+import org.hl7.cql.model.DataType
+
+class OperatorEntry(val name: String) {
```

Comment:
Are we not guarding against empty name Strings as the Java code does?

Reply:
I'll add it back.